src/components/grape-input/GrapeInput.js

```diff
@@ -66,7 +66,10 @@ export default class GrapeInput extends Component {
       this.props.channel.id &&
       nextProps.channel.id
     ) {
-      this.onChange({ value: nextProps.content })
+      // When a user switches to another channel the input changes the content,
+      // but the user actually didn't made any change to the input and therefor
+      // an update even triggering typing should be avoided
+      this.onChange({ value: nextProps.content, channelChanged: false })
```

Comment:
`channelChanged` should be true here and false in the other case. This then also need to properly handled in GrapeBrowser.